README.md

```diff
@@ -1,2 +1,157 @@
-# TeamRemi
+This framework abstracts the communication to an eventbus. Right now the only implemented eventbus is RabbitMQ. There are plans to support more eventbus implementations in the future.
```

Comment:
This framework is an abstraction layer for eventbusses. As of now the only implementation is for RabbitMq. There are plans to support more eventbusses in the future.